src/core/webview/ClineProvider.ts

```diff
@@ -96,6 +98,7 @@ type GlobalStateKey =
 	| "soundEnabled"
 	| "soundVolume"
 	| "diffEnabled"
+	| "checkpointsEnabled"
```

Comment:
Thinking that we change this to checkpointsDisabled or something when we're ready for prime time?

Reply:
Yes, I think that will work.